pkg/qdsa/patches/0001-fix-for-illegal-opcode-jmp-for-mcu-avr4-error.patch

```diff
@@ -0,0 +1,40 @@
+From f78cc5aff906cb133844b1ccd7f0300b3f3c1c49 Mon Sep 17 00:00:00 2001
+From: Hugues Larrive <hlarrive@pm.me>
+Date: Thu, 6 Jul 2023 18:10:09 +0200
+Subject: [PATCH] fix for illegal opcode jmp for mcu avr4 error
+
+---
+ avr/asm/keccak.S | 8 ++++++++
+ 1 file changed, 8 insertions(+)
+
+diff --git a/avr/asm/keccak.S b/avr/asm/keccak.S
+index 975b390..b1b4767 100644
+--- a/avr/asm/keccak.S
++++ b/avr/asm/keccak.S
+@@ -144,7 +144,11 @@ ch_XorLanesLoop:
+ 	push	rY+1
+ 	push	rInLen
+ 	push	rInLen+1
++#if (FLASHEND <= 0x1FFF)
++	rcall  KeccakF
++#else
+ 	call  KeccakF
++#endif
+ 	pop		rInLen+1
+ 	pop		rInLen
+ 	pop		rY+1
+@@ -195,7 +199,11 @@ ch_Padding:
+ 	subi	rT1,					0x80
+ 	st		Z,						rT1
+ 
++#if (FLASHEND <= 0x1FFF)
++	rcall  KeccakF
++#else
+ 	call  KeccakF
++#endif
+ 
+ 	//output 
+ 	ldi		rT3,			 		crypto_hash_BYTES/4			; copy 4 bytes per iteration
+-- 
+2.30.2
+
```

Comment:
Since we maintain that pkg, want to create a PR in https://github.com/RIOT-OS/qdsa directly?

Reply:
https://github.com/RIOT-OS/qDSA/pull/4

Reply:
You want to bump the commit hash of the pkg now